CI/build-mudlet-for-windows.sh

```diff
@@ -140,11 +140,6 @@ else
     export WITH_UPDATER="YES"
 fi
 
-# This one is VITAL as some things in the code have to be tweaked to be
-# different compared to the CI/CB build environment (or the
-# setup-windows-sdk.ps) one!
-export WITH_MAIN_BUILD_SYSTEM="NO"
```

Comment:
The "Main Build System" referred to was the old Mingw base one and that is long gone.